File: 26. Remove Duplicates from Sorted Array Easy/solution.py
def remove_duplicates(nums: list[int]) -> int:
    count = 0
    for i in range(len(nums)):
        if i < len(nums) - 1 and nums[i] == nums[i + 1]:
            continue
        nums[count] = nums[i]
        count += 1
    return count


print(remove_duplicates([1,1,2]))
print(remove_duplicates([0,0,1,1,1,2,2,3,3,4]))

File: 26. Remove Duplicates from Sorted Array Easy/solution.ts
function removeDuplicates(nums: number[]): number {
    let count = 0;
    for (let i = 0; i < nums.length; i++) {
        if (i < nums.length - 1 && nums[i] == nums[i + 1]) {continue}

        nums[count] = nums[i];
        count++;
    }
    return count;
}

console.log(removeDuplicates([1,1,2]))
console.log(removeDuplicates([0,0,1,1,1,2,2,3,3,4]))